plugins/chat/app/services/chat/publisher.rb

```diff
@@ -16,7 +20,7 @@ def self.publish_new!(chat_channel, chat_message, staged_id)
       content[:staged_id] = staged_id
       permissions = permissions(chat_channel)
 
-      MessageBus.publish("/chat/#{chat_channel.id}", content.as_json, permissions)
+      MessageBus.publish(root_message_bus_channel(chat_channel.id), content.as_json, permissions)
```

Comment:
Small one but I see this called six times. Not sure if it's worth an extension to message bus. 

Also, I'm guessing `root_message_bus_channel` is `message_bus_chat_channel_channel` 😅 Sorry for being pedantic since I'd imagine `root_message_bus_channel` is like the channel for all of chat.